### DISCUSSION TITLE
Adding Frontend to WezTerm

### DISCUSSION THREAD

**Discussion by ThePerfectComputer** [wezterm/wezterm#6046] [General]
I'd like to use WezTerm in a Rust web app I'm writing. The backend of the Rust WebApp would connect to a PTY/TTY, and the frontend(also Rust WASM) would spit characters into a monospace Text box. The text box can also listen for key events...

How doable is this or am I out of my mind?

If it's doable - where should I start poking in the code?

**Comment by Rezanmonis** [comment]
We are looking for frontend and backend developers as our teammates for our edtech project rezanmonis@gmail.com
